### ISSUE TITLE
[Request] External bat command support for code highlighting

### ISSUE THREAD

**Issue by jtplaarj** [mfontanini/presenterm#388]
I have added a PR (#387) for `tcl` and `verilog` support but it feels that this per language approach can be difficult. Apart from that, non main `bat` languages (`vhdl`, for example) are not easily supported. Much less locally downloaded syntax highlight files.

An alternative could be to provide a local `bat` command to `presenterm` to add flexibility.

**Comment by mfontanini** [comment]
You can already do something for this:

~~~markdown
```bash +exec_replace
bat --color always script.py
```
~~~

Does that work?

**Comment by jtplaarj** [comment]
Yes, that works great!!! I have used an slightly different approach, basically so that I do not need external files:

```md
```bash +exec_replace
echo '1b0:       00400793                li      a5,4
1e4:       50078513                add     a0,a5,1280 # fffff500 <__crt0_ram_last+0x7fffd501>
1e8:       2c9000ef                jal     cb0 <neorv32_uart_printf>' | batcat -p --color always -l asm 
```
```

I have also added the `-p` to avoid line number to better match de presenterm defaults. I am trying to make pandoc slides and presenterm compatible. I have an small perl script to translate from one to the other, I will try to add this as well.

Thanks!!
